### ISSUE TITLE
enable_debugger no longer working

### ISSUE THREAD

**Issue by taviso** [frida/frida#1667]
I think `session.enable_debugger()` is broken again, I just updated frida with pip and this no longer works:

```
    session = frida.attach(target_process)
    session.enable_debugger()
```

If I connect with devtools, it's just unresponsive. This script used to work just fine a few versions ago.

I guess I must be the only person who uses this, but I don't understand why, it's so great! Is the feature deprecated?

**Comment by taviso** [comment]
I have a [tiny utility](https://github.com/taviso/cefdebug) that can connect to a node debugger, I wonder if it would be possible to make a unittest that can detect breakage.

**Comment by oleavr** [comment]
@taviso Oh snap, sorry about that. (This feature is definitely not deprecated.)

Last time I went down the rabbit-hole to debug this, it turned out to be a regression in Chrome stable—87 at the time—which I verified by testing with Chrome Canary, where the debugger feature worked perfectly (wish I'd written down the Chrome bug number).

Recently a friend ran into the same issue with the latest Frida, and they confirmed that using Canary resolved the issue. However, since then Chrome 89 has become the stable version, so this is either a new regression in Chrome (highly unlikely) or a protocol interop issue (likely) with our [implementation](https://github.com/frida/frida-gum/blob/master/bindings/gumjs/guminspectorserver.c). Perhaps we've had an issue there since the beginning, and just got lucky up until Chrome's implementation changed.

I'm snowed under at the minute, but will try to get to this soon – I also really like this feature. Thanks so much for reporting!
